Backend/routes/routes.py

```diff
@@ -288,13 +301,75 @@ def update_supplementary(supplementary_id: str, supplementary: Supplementary):
 
 
 # Delete a Supplementary detail
+
+
 @supp_router.delete('/supplementary/{supplementary_id}')
 def delete_supplementary(supplementary_id: str):
-    result = SupplementaryCollection.delete_one({'_id': ObjectId(supplementary_id)})
+    result = SupplementaryCollection.delete_one(
+        {'_id': ObjectId(supplementary_id)})
     if result.deleted_count == 1:
         return {'message': 'Supplementary detail deleted successfully'}
     else:
-        raise HTTPException(status_code=404, detail='Supplementary detail not found')
+        raise HTTPException(
+            status_code=404, detail='Supplementary detail not found')
+
+
+@supplement_details.post('/add_supplement_details')
```

Comment:
from post to delete supplement details routes appearing two times(line no 317 to 372) it already present in line no  375 to 430.